medications/README.md

```diff
@@ -0,0 +1,152 @@
+| component / ehrSupplyPrescribe / quantity \[@unit]                                       | If present then: `MedicationRequest.dispenseRequest.quantity.value`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.extension`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.quantity.extension`.<br/>If none present then default value is `"Unk UoM"` (unknown unit of measurement).                                          |
+| component / ehrSupplyPrescribe / quantity / translation \[@value]                        | If present then: `MedicationRequest.dispenseRequest.quantity.value` otherwise defaults to `"1"`                                                                                                                                                                                                                                                                         |
+| component / ehrSupplyPrescribe / quantity / translation / originalText                   | If present then: `MedicationRequest.dispenseRequest.quantity.value`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.extension`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.quantity.extension`.<br/>If none present then default value is `"Unk UoM"` (unknown unit of measurement).                                          |
+| component / ehrSupplyPrescribe / inFulfillmentOf \[@typeCode]                            | fixed value = `"FLFS"`                                                                                                                                                                                                                                                                                                                                                  |
+| component / ehrSupplyPrescribe / inFulfillmentOf / priorMedicationRef \[@moodCode]       | fixed value = `"INT"`                                                                                                                                                                                                                                                                                                                                                   |
+| component / ehrSupplyPrescribe / inFulfillmentOf / priorMedicationRef / id \[@root]      | Built from `MedicationRequest.id` or if not present then new one generated                                                                                                                                                                                                                                                                                              |
```

Comment:
I think this is always generated, the MedicationRequest ID seems to be used to retrieve it